WordPress/src/main/java/org/wordpress/android/ui/themes/ThemeBrowserAdapter.java

```diff
@@ -65,6 +68,7 @@ public View newView(Context context, Cursor cursor, ViewGroup parent) {
 
         ThemeViewHolder themeViewHolder = new ThemeViewHolder(view);
         view.setTag(themeViewHolder);
+        mViewWidth = parent.getWidth();
```

Comment:
That's a good idea, and I did that partially.

Because the number of columns are figured out dynamically, I save the image width in preferences, and update it whenever it gets bigger. Worst case they have to download it twice total. After the rotation the max size is found and always used.